client/me/help/help-contact/index.jsx

```diff
@@ -501,9 +501,9 @@ const HelpContact = React.createClass( {
 		}
 	},
 
-	getContactFormCommonProps: function() {
+	getContactFormCommonProps: function( variationSlug ) {
 		const { olark, isSubmitting } = this.state;
-		const showHelpLanguagePrompt = ( olark.locale !== i18n.getLocaleSlug() );
+		const showHelpLanguagePrompt = ( olark.locale !== i18n.getLocaleSlug() ) && SUPPORT_FORUM !== variationSlug;
```

Comment:
Upon reading this, I am having this feeling that future maintainers who don't understand the background as much as we do could be confused that why we need to exclude the forum case specifically. It's a non-blocker, but I think adding some explanation here could be useful :)